versions.tf

```diff
@@ -11,5 +11,10 @@ terraform {
       source  = "hashicorp/helm"
       version = "~> 2.5"
     }
+
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.2.0"
```

Comment:
`version = "~> 3.2"` is better. This means it will include versions up to `3.999999` in the range, and `random` provider is rather stable, so there won't be surprises in minor versions.